common/services/allocation.test.js

```diff
@@ -667,13 +668,15 @@ describe('Allocation service', function () {
             fromLocations: [mockFromLocationId],
             toLocations: [mockToLocationId],
             locations: [mockFromLocationId, mockToLocationId],
+            include: ['profile', 'person'],
```

Comment:
This is just testing that the includes can be specified, right?

Might be a good idea not to use person as the value and use something obviously fake?